parm/gdas/aero_crtm_coeff.yaml

```diff
@@ -1,13 +1,13 @@
 mkdir:
 - $(DATA)/crtm/
 copy:
-- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/AerosolCoeff.bin, $(DATA)/crtm/]
-- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/CloudCoeff.bin, $(DATA)/crtm/]
-- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/v.viirs-m_npp.SpcCoeff.bin, $(DATA)/crtm/]
-- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/v.viirs-m_npp.TauCoeff.bin, $(DATA)/crtm/]
-- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/v.viirs-m_j1.SpcCoeff.bin, $(DATA)/crtm/]
-- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/v.viirs-m_j1.TauCoeff.bin, $(DATA)/crtm/]
-- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/NPOESS.VISice.EmisCoeff.bin, $(DATA)/crtm/]
-- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/NPOESS.VISland.EmisCoeff.bin, $(DATA)/crtm/]
-- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/NPOESS.VISsnow.EmisCoeff.bin, $(DATA)/crtm/]
-- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/NPOESS.VISwater.EmisCoeff.bin, $(DATA)/crtm/]
+- [$(CRTM_FIX)/AerosolCoeff.bin, $(DATA)/crtm/]
```

Comment:
I know we have discussed this before, but I would really like to move away from using `$( )` to `{{ }}`.

Reply:
Can/will do that now.

Reply:
Can you see any issue with the change/format in this PR? I'm getting an error:
`"expected <block end>, but found %r" % token.id, token.start_mark)`

Do they have to be in strings?

Reply:
Resolved. Figured out what the issue was.

Reply:
sorry was on a call.